app/src/main/java/org/fossasia/openevent/general/auth/LoginViewModel.kt

```diff
@@ -91,12 +91,12 @@ class LoginViewModel(
                 mutableRequestTokenSuccess.value = verifyMessage(it.message)
             }, {
                 mutableRequestTokenSuccess.value = verifyMessage(it.message.toString())
-                mutableError.value = resource.getString(R.string.email_not_in_server_message)
+                mutableError.value = resource.getString(R.string.reset_password_mail_message)
             })
     }
 
     private fun verifyMessage(message: String): Boolean {
-        if (message == resource.getString(R.string.email_sent)) {
+        if (message.replace(" ", "") == resource.getString(R.string.reset_password_mail_message)?.replace(" ", "") ?: "") {
```

Comment:
Show the message from server. This method is bound to fail whenever the message from server is changed

Reply:
That's why I was asking you but you said that it's the app's problem.
Response - 
![response](https://user-images.githubusercontent.com/31654207/70561539-a1256d00-1bb0-11ea-8d6f-4210b3e97caa.jpg)

Reply:
It is

Reply:
I'm saying that show the {"message"} from the server in the app. Don't match the message from the server. Just unconditionally show it

Reply:
I think there's no need of verifyMessage() function then.

Reply:
Yes